src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -83,29 +86,28 @@ public CaseDetails saveCountyCourtJudgment(
         CCDCase ccdCase = this.caseMapper.to(claim);
         ccdCase.setCountyCourtJudgment(countyCourtJudgmentMapper.to(countyCourtJudgment));
         ccdCase.setCountyCourtJudgmentRequestedAt(now().format(ISO_DATE_TIME));
-        return this.update(authorisation, ccdCase, DEFAULT_CCJ_REQUESTED);
+        return this.update(authorisation, ccdCase, DEFAULT_CCJ_REQUESTED, ccdCase.getSubmitterId());
     }
 
     public CaseDetails saveDefendantResponse(
-        String authorisation,
         Claim claim,
+        String defendantId,
         String defendantEmail,
         Response response,
-        String defendantId
+        String authorisation
     ) {
 
         CCDCase ccdCase = this.caseMapper.to(claim);
-        ccdCase.setSubmitterId(defendantId);
         ccdCase.setResponse(responseMapper.to((FullDefenceResponse) response));
         ccdCase.setDefendantEmail(defendantEmail);
         ccdCase.setRespondedAt(now().format(ISO_DATE_TIME));
-        return this.update(authorisation, ccdCase, DEFENCE_SUBMITTED);
+        return this.update(authorisation, ccdCase, DEFENCE_SUBMITTED, defendantId);
     }
 
-    public CaseDetails update(String authorisation, CCDCase ccdCase, CaseEvent caseEvent) {
+    public CaseDetails update(String authorisation, CCDCase ccdCase, CaseEvent caseEvent, String userId) {
```

Comment:
Don't need to pass userId as userId can be extracted from `authorisation` here like ```String userId = userService.getUserDetails(authorisation).getId();```